supportedBackends/google/pipelines/v1alpha2/src/main/scala/cromwell/backend/google/pipelines/v1alpha2/PipelinesConversions.scala

```diff
@@ -30,22 +31,38 @@ object PipelinesConversions {
   implicit val fileOutputTo: ToParameter[PipelinesApiFileOutput] = new ToParameter[PipelinesApiFileOutput] {
     override def toGooglePipelineParameter(output: PipelinesApiFileOutput): PipelineParameter = {
       new PipelineParameter().setName(output.name).setLocalCopy(
-        new LocalCopy().setDisk(output.mount.name).setPath(output.local.pathAsString)
+        new LocalCopy().setDisk(output.mount.name).setPath(output.relativeHostPath.pathAsString)
       )
     }
 
-    override def toGoogleRunParameter(output: PipelinesApiFileOutput): String = output.cloudPath
+    override def toGoogleRunParameter(output: PipelinesApiFileOutput): String = output.cloudPath.pathAsString
+  }
+
+  implicit val outputTo: ToParameter[PipelinesApiOutput] = new ToParameter[PipelinesApiOutput] {
+    override def toGooglePipelineParameter(output: PipelinesApiOutput): PipelineParameter = output match {
+      case fileOutput: PipelinesApiFileOutput => fileOutputTo.toGooglePipelineParameter(fileOutput)
+      case _: PipelinesApiDirectoryOutput =>
+        throw new UnsupportedOperationException("Pipelines API V1 backend does not support PipelinesApiDirectoryOutput parameters. This is a programmer error.")
```

Comment:
I don't think so, it would mean that the PAPI common code created a `PipelinesApiDirectoryOutput` which it does not right now, only the V2 specific code does that.
This is due to the mix and match between common PAPI code and version specific code. A LOT of this should be cleaner when we remove V1 support.